### PR TITLE
Mesh channels

### PR DESCRIPTION
**Pull request by heuperman**
Connect to your friends via Bluetooth, P2P WiFi, the works!

### CODE REVIEW

File: messenger-app.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -646,13 +652,14 @@
...
-				DEVELOPMENT_TEAM = 59DM9F2W6S;
+				DEVELOPMENT_TEAM = ZAS292B37D;
 				INFOPLIST_FILE = "messenger-app/Info.plist";
+				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
 				LD_RUNPATH_SEARCH_PATHS = (
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = "Messenger-Group.messenger-app";
+				PRODUCT_BUNDLE_IDENTIFIER = com.CornelisHeuperman.Meshenger;
```
Comment (neobay991) [inline messenger-app.xcodeproj/project.pbxproj]: Does this mean you need to make a change in firebase too?

File: messenger-app.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -668,13 +675,14 @@
...
```
Comment (neobay991) [inline messenger-app.xcodeproj/project.pbxproj]: same as previous comment

**Comment by YvCodeHong** [comment]
:shipit:

File: messenger-app.xcodeproj/project.pbxproj (hunk 1 above)
Comment (heuperman) [reply messenger-app.xcodeproj/project.pbxproj]: Yes, I do. It's working at the moment but does give a warning. I'll change it before we merge to master.

**Comment by neobay991** [comment]
:shipit:

**Pull request merged**